packages/grafana-data/src/utils/Registry.ts

```diff
@@ -127,6 +129,13 @@ export class Registry<T extends RegistryItem> {
     return this.ordered; // copy of everythign just in case
   }
 
+  isEmpty(): boolean {
+    if (!this.initialized) {
+      this.getIfExists('xxx'); // will trigger init
+    }
```

Comment:
Can we refactor this if statement to `private initialise` method? This looks funky;)